docs/source/sim_overview.rst

```diff
@@ -56,55 +54,28 @@ give you at least a basic exploratory idea of the performance of your review:
 	asreview plot myreview.h5
 	asreview plot DIR_WITH_MULTIPLE_SIMULATIONS
 
-	
+For an example of results of a simulation study, see the `Simulation Results <simulation-results.html>`__ page.
+
+
 Simulation options
 ------------------
-The options for simulating are shown on the `CLI <cli.html>`__ page. We will highlight some of the
-more often used options here.
+The options for simulating are shown on the `CLI <cli.html>`__ page. We will highlight some of the more often used options here. When no additional arguments are specified in the ``asreview simulate`` command, default settings are used.
+
+To make your simulations reproducible you can use the ``--seed`` and ``--init_seed`` options. 'Init
+seed' controls the starting set of papers to train the model on, while the 'seed' controls the
+random generator after initialization.
 
-To make your simulations reproducible you can use the ``--seed`` and ``--init_seed`` options. Init
-seed controls the starting set of papers to train the model on. While the normal seed controls the
-random generator after initialization. For initialization you can also use ``--prior_idx`` to set
-the papers to start the simulation with.
+By default, the model initializes with one relevant and one irrelevant record. If you want to initialize your model with a specific set of starting papers, you can use ``--prior_idx`` to select the indices of the papers you want to start the simulation with.
```

Comment:
Would swap the order of this and the paragraph below. You can set the number of priors by ..., but if you want more control, use ...

Reply:
Which paragraph do you mean? The next paragraph is about `n_instances`, which is about the number of records to be labeled before the model retrains again. I've updated the current paragraph by adding the arguments `--n_prior_included` and `--n_prior_excluded`: 

> By default, the model initializes with one relevant and one irrelevant record. You can set the number of priors by `--n_prior_included` and `--n_prior_excluded`. However, if you want to initialize your model with a specific set of starting papers, you can use ``--prior_idx`` to select the indices of the papers you want to start the simulation with.

Reply:
Yes, that's what I meant.